Big-Spender/readme.md

```diff
@@ -68,39 +68,39 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends WHERE description ILIKE '%fee%' OR description ILIKE '%fees%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends s JOIN expense_areas e ON (s.expense_area_id = e.id) WHERE e.expense_area = 'Better Hospital Food';
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+SELECT to_char(date, 'YYYY-MM') AS month, sum(amount) FROM spends GROUP BY month;
 ```
 
 **Farnoosh:** Thanks, that's really useful. We also need to know the total amount spent on each supplier. Can you help us with that?
 
 **You:** Sure thing. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT supplier_id, sum(amount) AS total FROM spends GROUP BY supplier_id;
 ```
 
 **Farnoosh:** Oh, how do I know who these suppliers are? There's only numbers here.
 
 **You:** Whoops! I gave you ids to key the totals, but let me give you names instead.
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT supplier_id, supplier, sum(amount) AS total FROM spends s JOIN suppliers sp ON (s.supplier_id = sp.id) GROUP BY supplier_id, supplier;
```

Comment:
yep, exactly. This looks very good.

Reply:
Thanks for review Karley , i made some changes :)